docs/content/authentication.md

```diff
@@ -5,13 +5,97 @@ title: "Authentication"
 
 # Authentication and Configuration
 
-Each system handles authentication differently. Authentication must be configured
-if resources are provided for a system.
+Authentication for Grizzly can be configured in two ways: using environment variables
+and through Grizzly's own 'context' support (much like `kubectl` contexts).
+
+Environment variables are suitable for use against a single system, and best suited
+to CI and automation scenarios.
+
+Grizzly contexts allow easy switching between multiple configurations and are best
+suited to workstation use.
+
+# Using Grizzly Contexts
+
+By default, the `default` context is used. We will discuss how to use the default
+context first, then explain how to use multiple contexts thereafter.
+
+> **NOTE**: If you have used Grizzly previously with environment variables, you can
+  initialise the `default` context from your environment simply with `grr config import`.
```

Comment:
So, Grizzly does not work without using that command first? Isn't that a breaking change?

Reply:
No, envvars override contexts. So this is a way to transfer your envvars. What we need to document is that you then need to *unset* your envvars otherwise they will continue to override.